### PR TITLE
Header/michellec

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description:
- Shifted the side drawer down
- Side drawer closes to show icons even when closed
- Styled new header component and made to fit full width
- Import Lato to apply font globally to all MUI components

## Checklist
- [x]  Size of pull request kept to a minimum
- [x]  Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by katieolson84 — CHANGES_REQUESTED** [review]
Great job using createMuiTheme and ThemeProvider to customize the font to match the site. I did notice that you self-imported the font into the code. The font has already been imported into the HTML. Can you explain why you structured it the way you did?

We mentioned switching out the icons from MUI to the ones in the styles folder. Let me know if you need help with that process.

The rest of your code looks great.

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 1]
```diff
@@ -125,57 +96,30 @@ export default function SideDrawer(props) {
...
-          <IconButton onClick={handleDrawerClose}>
-            {theme.direction === 'ltr' ? (
-              <ChevronLeftIcon />
-            ) : (
-              <ChevronRightIcon />
-            )}
+        <div className={classes.toolbar}>
+          <IconButton onClick={handleDrawer}>
+            {open === false ? <ChevronRightIcon /> : <ChevronLeftIcon />}
+            {/* {open === false ? <ArrowRight /> : <ArrowLeft />} */}
```
Comment (katieolson84) [inline src/components/pages/SideDrawer/SideDrawer.js:122]: Did you want to leave this commented code in?

Comment (MichelleCCodes) [reply src/components/pages/SideDrawer/SideDrawer.js:122]: Yes! I wanted to get back to this tomorrow when I try importing the icons. Thanks for asking!

**Comment by MichelleCCodes** [comment]
The .woff2 file and self-imported Lato font was required to customize fonts of MUI components. Without the file and new theme, MUI components would remain unaffected by the global font that was imported in HTML file. All MUI components should inherit this new font now, since the new theme was applied at the root of the app! 🙏

File: src/components/pages/SideDrawer/SideDrawer.js (hunk 1 above)
Comment (katieolson84) [reply src/components/pages/SideDrawer/SideDrawer.js]: Perfect!

**Comment by katieolson84** [comment]
Thank you for explaining the reason for the self-imported font, that makes sense and it will be great to have the font globally through MUI. You are great!

**Review by katieolson84 — APPROVED** [review]
This looks great. Thank you for clarifying your code from the previous review.

**Review by WeiWu2 — APPROVED** [review]
I see no problems here. Great job!

**Pull request merged**